pkgs/applications/networking/cluster/prow/default.nix

```diff
@@ -2,41 +2,39 @@
 
 buildGoModule rec {
   pname = "prow-unstable";
-  version = "2019-08-14";
-  rev = "35a7744f5737bbc1c4e1256a9c9c5ad135c650e4";
+  version = "2020-03-26";
+  rev = "b51b6f39db5d29213a72bfbac238f41c9b24462a";
```

Comment:
This was already done in #84055. You should probably rebase.

Reply:
Should be sorted now.